app/uk/gov/hmrc/merchandiseinbaggage/service/CalculationService.scala

```diff
@@ -69,7 +69,8 @@ class CalculationService @Inject()(mibConnector: MibConnector)(implicit ec: Exec
 
   def thresholdAllowance(declaration: Declaration)(implicit hc: HeaderCarrier): Future[ThresholdAllowance] = {
     import declaration._
-    paymentCalculations(paidAndNotRequired(amendments), goodsDestination).map(calculation =>
+    val totalGoods = declarationGoods.goods ++ paidAndNotRequired(amendments)
```

Comment:
This will not work for `Export` as `paid` and `NotRequired` are never be set for amendments

Reply:
ok...will do it now

Reply:
yeah, if you can re-name and re-use `object DeclarationView` logic 

```  private def validAmendments(declaration: Declaration) =
    declaration.declarationType match {
      case Export => declaration.amendments
      case Import => declaration.amendments.filter(a => a.paymentStatus.contains(Paid) || a.paymentStatus.contains(NotRequired))
    }```